site/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="robots" content="noindex" />
-    <title>kpt - Home</title>
+    <title>Home</title>

```

Comment:
for seo, you probably want to have "kpt" in the title, not sure
I understand the repetition is annoying

Reply:
Let's put it back in for now and we'll see if there's any feedback on it.